basyx.aasenvironment/basyx.aasenvironment-http/src/test/java/org/eclipse/basyx/digitaltwin/aasenvironment/http/TestAasEnvironmentHTTP.java

```diff
@@ -140,8 +162,8 @@ public String readStringFromFile(String fileName) throws FileNotFoundException,
 		return IOUtils.toString(in, StandardCharsets.UTF_8.name());
 	}
 
-	@AfterClass
-	public static void shutdown() {
+	@After
```

Comment:
Why did you change this to @After?

Reply:
Reverted this to the previous state i.e. @AfterClass